README.md

```diff
@@ -25,16 +25,20 @@ import { EntityDXDashboardContent } from '@get-dx/backstage-plugin';
 
 const serviceEntityPage = (
   <EntityLayout>
+    {/* ... */}
+
     <EntityLayout.Route path="/dx" title="DX">
       <EntityDXDashboardContent />
     </EntityLayout.Route>
+
+    {/* ... */}
   </EntityLayout>
 )
 ```
 
 > See the Components section below for other components offered.
 
-## Components
+## DX Core Components
```

Comment:
Thoughts on this section? I'm unsure of what we want to describe these components

Reply:
Just colloquially "Core". Good point that it might lead folks to think it's just about Core4. 🤔